src/twisted/web/test/test_http.py

```diff
@@ -3325,6 +3325,41 @@ def test_expect100ContinueHeader(self):
 
 
 
+class ConnectRequestTests(unittest.TestCase, ResponseTestMixin):
+    """
+    Test correct RFC 7231 handling for CONNECT requests. Responses to these
```

Comment:
You can leave off "correct" here.  Tests are always for correct behavior. :wink: